src/scss/global.scss

```diff
@@ -772,3 +772,9 @@ span.link {
 .flex-one {
   flex: 1;
 }
+
+.no-outline-on-focus {
+  &:focus {
+    outline: none;
+  }
+}
```

Comment:
Why was this added? Removing outline on focus creates accessibility issues, not fixes them..

Reply:
removed